clang/test/CodeGen/cheri/cheri-asm-ptrtoint.c

```diff
@@ -0,0 +1,31 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 2
+// RUN: %riscv32_cheri_purecap_cc1 -S -emit-llvm -O1 -o - %s -verify | FileCheck %s --check-prefix=RV32
+// RUN: %riscv64_cheri_purecap_cc1 -S -emit-llvm -O1 -o - %s -verify | FileCheck %s --check-prefix=RV64
```

Comment:
Why -O1? Is this just a case of wanting to run mem2reg? If so, do that.

Reply:
:+1: